Jenkinsfile

```diff
@@ -4,8 +4,42 @@ properties(
   ]
 )
 
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
 node('buildvm-devops') {
-  sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
-    echo "Bruno's branch split pipeline"
+  try {
+    deleteDir()
+    set_workspace()
+    stage('clone') {
+      dir('aos-cd-jobs') {
+        checkout scm
```

Comment:
No, we have to explicitly tell jenkins to clone the repository.  I guess that's
because you might not want to.

What we do get with the scm trigger is the ability to clone it without giving
the source url, which is nice for testing with a local mirror.